packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -113,17 +101,36 @@ const Encryption = WebexPlugin.extend({
       }
     })
       .then((res) => {
-        const url = res.body.endpoints[scr.loc];
+        const url = scr.loc;
 
-        if (!url) {
-          this.logger.warn('encryption: could not determine download url for `scr.loc`; attempting to download `scr.loc` directly');
+        return _this2._callDownloadUrl(url, scr, res);
+      });
+  },
 
-          return scr.loc;
-        }
-        this.logger.info('encryption: retrieved download url for encrypted file');
+  _callDownloadUrl(url, scr, result) {
+    const _this2 = this;
 
-        return url;
-      });
+    return this.request({
+      method: 'GET',
+      uri: url
+    }).then(() => _this2._helperFunction(scr, result))
+      .catch((err) => Promise.reject(err));
+  },
+
+  _helperFunction: function _helperFunction(scr, result) {
```

Comment:
It appears that JSDoc is missing from this method, take a look at [this file](https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/webex-core/src/lib/services/services.js#L55) for reference.